Zend/zend_exceptions.c

```diff
@@ -867,6 +868,10 @@ void zend_register_default_exception(void) /* {{{ */
 	INIT_CLASS_ENTRY(ce, "DivisionByZeroError", NULL);
 	zend_ce_division_by_zero_error = zend_register_internal_class_ex(&ce, zend_ce_arithmetic_error);
 	zend_ce_division_by_zero_error->create_object = zend_default_exception_new;
+
+	INIT_CLASS_ENTRY(ce, "UnhandledMatchError", NULL);
```

Comment:
I think it would be more consistent in PHP for an internal **class inheriting from `\Exception` (`zend_ce_exception`) to be called `UnhandledMatchException`**

For example, everything in https://www.php.net/manual/en/spl.exceptions.php, and ErrorException, both is a descendent of Exception and has a suffix of Exception.

If you did intend for it to inherit from Error, the class entry for error is zend_ce_error, but I personally don't think it's that Severe.

Reply:
Oops, yes I meant to inherit from `zend_ce_error`. Somebody in the mailing list mentioned that operations only throw errors.

https://externals.io/message/109383#109400